docs/intro/install.rst

```diff
@@ -118,27 +119,24 @@ Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
 
   conda install -c conda-forge scrapy
 
-To install Scrapy on Windows using ``pip`` :-
-
-1. Install latest version of `Python`_ (Recommended to tick [Add Python to PATH] checkbox during installation)
-
-2. Install or upgrade pip::
+To install Scrapy on Windows using ``pip`` :
 
-    python -m pip install --upgrade pip
+.. warning::
+    This installation method requires “Microsoft Visual C++” for installing some 
+    scrapy dependencies which demands significantly more disk space than Anaconda.
 
-3. Create Venv (Virtual Environment)::
+#. Download and execute `Microsoft C++ Build Tools`_ to install the Visual Studio Installer.
 
-    python -m venv [env name]
+#. Open Visual Studio Installer, install Visual Studio and modify installation.
 
-4. Download `Microsoft Visual C++ 14.0`_
+#. Under workloads section, select **Desktop development with C++**.
 
-5. Run the downloaded EXE file and let it setup the Visual Studio Installer
+#. Below Installation details, select and install following packages :
 
-6. Open Visual Studio Installer, select and install C++ build tools
+  * **MSVC**  (e.g MSVC v142 - VS 2019 C++ x64/x86 build tools (v14.23) )
+  * **Windows SDK**  (e.g Windows 10 SDK (10.0.18362.0))
```

Comment:
Do you need to modify the installation? Can’t you install all that is needed at the same time as you install Visual Studio?

I’m thinking that we could replace these steps by indicating to the user to install Visual Studio (linking to the upstream documentation, https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio ) and indicate which workloads and individual components need to be selected during that process. I assume MSVC and Windows SDK are listed under “Individual components”.

Reply:
> Do you need to modify the installation? Can’t you install all that is needed at the same time as you install Visual Studio?

modification window automatically pops up(including workloads and individual components) when we install visual studio.

Reply:
> I’m thinking that we could replace these steps by indicating to the user to install Visual Studio (linking to the upstream documentation, https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio )

That's a good idea. Linking to upstream documentation will help user understanding the workflows.



>  indicate which workloads and individual components need to be selected during that process. I assume MSVC and Windows SDK are listed under “Individual components”.

under workloads when we select **Desktop development with C++** . Those 2 packages are already selected along with some other packages in option components section on the right side. Maybe we should just tell users make sure MSVC and SDK are selected.